client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -40,13 +40,13 @@
 
 <div class="field">
     <label translate>Dateline</label>
-    <div sd-meta-locators class="dateline-info" ng-show="_editable && !proofread"
+    <div sd-meta-locators class="dateline-info" ng-show="_editable"
```

Comment:
Any specific reason for removing proofread here? In proofread mode we should show the dateline.text not the dateline fields.